### ISSUE TITLE
Unknown error

### ISSUE THREAD

**Issue by aronchick** [bacalhau-project/bacalhau#2952]
I think it's either nothing matches the label, or no networking. But error is not helpful.

----------

```

❯ cat job.yaml | bacalhau create
Reading from /dev/stdin; send Ctrl-d to stop.Job successfully submitted. Job ID: 7a577139-80d8-4da4-bd02-cca0a70df9c8
Checking job status... (Enter Ctrl+C to exit at any time, your job will continue running):

	Communicating with the network  ................  done ✅  0.1s
	   Creating job for submission  ................  err  ❌  0.0s

Error submitting job: not enough nodes to run job. requested: 1, available: 0.
Job Results By Node:

To get more details about the run, execute:
	bacalhau describe 7a577139-80d8-4da4-bd02-cca0a70df9c8
❯ bacalhau describe 7a577139-80d8-4da4-bd02-cca0a70df9c8
Job:
  APIVersion: V1beta2
  Metadata:
    ClientID: 3aa6bb9e9bd5ed21781a1f16cf1f45f01b3f139ed003aa8ba0a6e486a151ca0f
    CreatedAt: "2023-10-25T17:26:49.473778378Z"
    ID: 7a577139-80d8-4da4-bd02-cca0a70df9c8
    Requester:
      RequesterNodeID: QmdZQ7ZbhnvWY1J12XYKGHApJ6aufKyLNSvf8jZBrBaAVL
      RequesterPublicKey: CAASpgIwggEiMA0GCSqGSIb3DQEBAQUAA4IBDwAwggEKAoIBAQDVRKPgCfY2fgfrkHkFjeWcqno+MDpmp8DgVaY672BqJl/dZFNU9lBg2P8Znh8OTtHPPBUBk566vU3KchjW7m3uK4OudXrYEfSfEPnCGmL6GuLiZjLf+eXGEez7qPaoYqo06gD8ROdD8VVse27E96LlrpD1xKshHhqQTxKoq1y6Rx4DpbkSt966BumovWJ70w+Nt9ZkPPydRCxVnyWS1khECFQxp5Ep3NbbKtxHNX5HeULzXN5q0EQO39UN6iBhiI34eZkH7PoAm3Vk5xns//FjTAvQw6wZUu8LwvZTaihs+upx2zZysq6CEBKoeNZqed9+Tf+qHow0P5pxmiu+or+DAgMBAAE=
  Spec:
    Deal:
      Concurrency: 1
    Docker: {}
    EngineSpec:
      Params:
        Entrypoint:
        - /bin/bash
        - -c
        - python3 /process.py /var/log/logs_to_process/aperitivo_logs.log.1 "SELECT
          * FROM log_data WHERE message LIKE '%[SECURITY]%' ORDER BY '@timestamp'"
        EnvironmentVariables: null
        Image: docker.io/bacalhauproject/motherduck-log-processor:1.1.1
        Parameters: null
        WorkingDirectory: ""
      Type: docker
    Inputs:
    - Name: file:///var/log/logs_to_process
      Path: /var/log/logs_to_process
      StorageSource: localDirectory
    - Name: file:///db/
      Path: /db
      StorageSource: localDirectory
    Network:
      Type: Full
    NodeSelectors:
    - Key: zone
      Operator: =
      Values:
      - europe-west4-b
    PublisherSpec:
      Type: ipfs
    Resources:
      GPU: ""
    Timeout: 1800
    Wasm:
      EntryModule: {}
State:
  CreateTime: "2023-10-25T17:26:49.473778378Z"
  Executions: []
  JobID: 7a577139-80d8-4da4-bd02-cca0a70df9c8
  State: Error
  TimeoutAt: "2023-10-25T17:56:49.473778378Z"
  UpdateTime: "2023-10-25T17:26:49.483037476Z"
  Version: 2

```

**Comment by aronchick** [comment]
Source: I accidentally scheduled against the public network which didn't have any nodes that matched the label.

**Comment by aronchick** [comment]
Could also be "didn't have access to LocalDir"

**Comment by aronchick** [comment]
Feel free to expand this bug to also take on what more information we can provide during regular job running.

**Comment by aronchick** [comment]
As part of this, you should say what the IP address/DNS address you are submitting to

**Comment by aronchick** [comment]
See if outputting the labels on the requestor node is valuable?

**Comment by simonwo** [comment]
Resolved by #3008.
